NeuroID/NeuroIDClass/Extensions/NIDSend.swift

```diff
@@ -95,8 +81,8 @@ public extension NeuroID {
             headers: headers
         ).responseData { response in
             completion(response)
-            
-            if response.error != nil && retryCount > 0 {
+
+            if response.error != nil, retryCount > 0 {
```

Comment:
I have a prettier like extension in my Xcode that auto changes things into the native swift equivalent, I didn't even notice because I wrote `&&`. Should I try to change it back?